config/data_scrubber.go

```diff
@@ -35,23 +37,48 @@ func NewDefaultDataScrubber() *DataScrubber {
 
 // CompileStringsToRegex compile each word in the slice into a regex pattern to match
 // against the cmdline arguments
-// The word must contain only word characters ([a-zA-z0-9_])
+// The word must contain only word characters ([a-zA-z0-9_]) or wildcards *
 func CompileStringsToRegex(words []string) []*regexp.Regexp {
 	compiledRegexps := make([]*regexp.Regexp, 0, len(words))
-	forbiddenSymbols := regexp.MustCompile(`\W`)
+	forbiddenSymbols := regexp.MustCompile("[^a-zA-Z0-9_*]")
 
 	for _, word := range words {
 		if forbiddenSymbols.MatchString(word) {
-			log.Errorf("warning data scrubber - %s not compiled. The sensitive word must contain only word characters", word)
+			log.Warnf("data scrubber - %s not compiled. The sensitive word must "+
```

Comment:
last thing: can you switch this and the other line to `data scrubber: ...` to match go style?